lib/graphql_devise/util/class_getter.rb

```diff
@@ -0,0 +1,11 @@
+module GraphqlDevise
+  module Util
+    module ClassGetter
```

Comment:
I think this class is very similar to [safe_constantize](https://apidock.com/rails/ActiveSupport/Inflector/safe_constantize). WDYT about using this instead?

Reply:
🤦‍♂